tools/merger/src/ci_generate.js

```diff
@@ -5,7 +5,7 @@ const Mustache = require("mustache");
 const { allPackagePaths, allPluginPaths } = require("../../shared/packages");
 const execSync = require("../../shared/exec");
 
-const docker_image = "circleci/node:16.8.0";
+const docker_image = "cimg/node:16.14";
```

Comment:
👍 There was nothing special about the node image we were using... at one point, it was the supported one.